controllers/articles.controllers.js

```diff
@@ -0,0 +1,26 @@
+const {
+    fetchArticleByID
+} = require("../models/articles.models");
+
+/**@type {import("express").RequestHandler} */
+module.exports.getArticleByID = async (req, res, next) => {
+    const { article_id } = req.params;
+    
+    try
+    {
+        const article = await fetchArticleByID(article_id);
+
+        if (!article)
+        {
+            res.status(404).send({msg: "Article not found"});
+        }
+        else
```

Comment:
the if else statement almost defeats the purpose of the try catch block. have a think about how you could refactor this so be cleaner